src/Catty/PlayerEngine/DataModel/CBSpriteNode/CBSpriteNode.swift

```diff
@@ -43,7 +43,18 @@ class CBSpriteNode: SKSpriteNode {
     @objc required init(spriteObject: SpriteObject) {
         let color = UIColor.clear
         self.spriteObject = spriteObject
-        self.penConfiguration = PenConfiguration()
+
+        if let project = self.spriteObject.project {
+            if let width = project.header.screenWidth, let height = project.header.screenHeight {
+
+                self.penConfiguration = PenConfiguration(projectHeaderScreenWidth: CGFloat(truncating: width), projectHeaderScreenHeight: CGFloat(truncating: height))
```

Comment:
Please add a test to `CBSpriteNodeTests` to verify that the `PenConfiguration` is receiving the proper width and height.